code/modules/hydroponics/seeds.dm

```diff
@@ -1339,6 +1339,10 @@
 	if(parent)
 		mutations = parent.mutations
 
+/obj/item/seeds/kudzuseed/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] swallows the pack of kudzu seeds! It looks like \he's trying to commit suicide..</span>")

```

Comment:
might as well plant it, you already have the seed in order to use it so it's not like it's exploitable.